tests/Form/Type/AdminTypeTest.php

```diff
@@ -225,12 +241,15 @@ public function testArrayCollectionNotFound(): void
     protected function getExtensions()
     {
         $extensions = parent::getExtensions();
+
         $guesser = $this->prophesize(FormTypeGuesserInterface::class)->reveal();
         $extension = new TestExtension($guesser);
 
         $extension->addTypeExtension(new FormTypeFieldExtension([], []));
         $extensions[] = $extension;
 
+        $extensions[] = new PreloadedExtension([$this->adminType], []);
+
         return $extensions;
     }
 }
```

Comment:
Please copy paste the unmodified test file as `LegacyAdminTypeTest` and add assertions on the deprecation message.